client/src/components/TaskList.tsx

```diff
@@ -54,7 +54,7 @@ export const TaskList: React.FC<any> = ({ tasks }) => {
     <>
       <IonList>
         {
-          tasks.map((task : ITask) => {
+          tasks.map((task: ITask) => {
```

Comment:
And this needs to be `tasks.items.map` or we pass down `tasks.items` from the `TaskPage` component instead.

Reply:
Actually, I think it will be better to pass down the `tasks.items` as the prop instead and make the change in the `TaskPage` component. Will look at this after I figure out the build issue

Reply:
Yep. This is the missing part I did not had time to fix.